tests/test_series.py

```diff
@@ -274,7 +295,119 @@ def test_monograph(app):
+            </datafield>
+            <datafield tag="245" ind1=" " ind2=" ">
+                <subfield code="a">Wissenschaftliche Abhandlungen</subfield>
+            </datafield>
+            <datafield tag="246" ind1=" " ind2=" ">
+                <subfield code="n">v.1</subfield>
+                <subfield code="p">1865-1874</subfield>
+            </datafield>
+            <datafield tag="246" ind1=" " ind2=" ">
+                <subfield code="n">v.2</subfield>
+                <subfield code="p">1875-1881</subfield>
+            </datafield>
+            <datafield tag="246" ind1=" " ind2=" ">
+                <subfield code="n">v.3</subfield>
+                <subfield code="p">1882-1905</subfield>
+            </datafield>
+            <datafield tag="300" ind1=" " ind2=" ">
+                <subfield code="a">3 v</subfield>
+            </datafield>
+            """,
+            {
+                '_migration': {'volumes': [
+                    {
+                        'volume': 3,
+                        'isbn': '1108052819',
+                        'physical_description':
+                            'print version, paperback',
+                    },
+                    {
+                        'volume': 3,
+                        'isbn': '9781108052818',
+                        'physical_description':
+                            'print version, paperback',
+                    },
+                    {
+                        'volume': 2,
+                        'isbn': '9781108052801',
+                        'physical_description':
+                            'print version, paperback',
+                    },
+                    {
+                        'volume': 2,
+                        'isbn': '1108052800',
+                        'physical_description':
+                            'print version, paperback',
+                    },
+                    {
+                        'volume': 1,
+                        'isbn': '9781108052795',
+                        'physical_description':
+                            'print version, paperback',
+                    },
+                    {
+                        'volume': 1,
+                        'isbn': '1108052797',
+                        'physical_description':
+                            'print version, paperback',
+                    },
+                    {
+                        'title': '1865-1874',
+                        'volume': 1
+                    },
+                    {
+                        'title': '1875-1881',
+                        'volume': 2
+                    },
+                    {
+                        'title': '1882-1905',
+                        'volume': 3
+                    },
+                ],
+                    'record_type': 'multipart',
+                },
+                "title": [{'title': 'Wissenschaftliche Abhandlungen'}],
```

Comment:
shouldn't it be:
```
title: {
    title: ''
}
```
?
is it a list?

Reply:
ah, it is a leftover from serials (they can have list of titles in temporary files as book might be a part of many serials) I will fix it for multiparts